coderadar-test/src/main/asciidoc/restapi.adoc

```diff
@@ -121,6 +122,8 @@ If the log in is successful, user obtains two JSON Web Tokens https://jwt.io
 
 How to use the tokens is described in <<authentication, Authentication>>
 
+// TODO shouldn't it be a hash which is sent to the server?
```

Comment:
If you're talking about the password, then only the server does the hashing, doing so in the front-end would require that we use the same algorithm as  in the backend.  Or do you propose hashing it twice? Once in the front-end and once in the back-end?

Reply:
sending encrypted password or the hash of the password both have advantages and disadvantages. sending it in clear publishes the password if encryption is cracked, sending it as a hash makes replay attacks possible if encryption is cracked..